realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProcessor.java

```diff
@@ -123,7 +124,9 @@
         "io.realm.annotations.RealmModule",
         "io.realm.annotations.Required"
 })
+@SupportedOptions(value = {"debug"})
```

Comment:
I think we can use `OPTION_DEBUG` constant instead of hard coded value `"debug"` like `@SupportedOptions(value = {RealmProxyClassGenerator.OPTION_DEBUG})`.